test.sh

```diff
@@ -32,7 +32,7 @@ TESTS=$(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...)
 go test -v ${TESTS}
 
 # Ignore these paths in the following tests.
-ignore="vendor\|\_gopath\|assets.go"
+ignore="vendor\|\_gopath\|\assets.go\|\constants_gendocs.go"
```

Comment:
@jimmidyson , actually I am getting some boilerplate related issue for `constants_gendocs.go` which I have no idea how to fix it.

```
Checking boilerplate...
Boilerplate missing in: ././pkg/minikube/constants/constants_gendocs.go.
Makefile:82: recipe for target 'test' failed
make: *** [test] Error 1
```
Any idea why I am getting such error?